pysac/mhs_atmosphere/hs_model/__init__.py

```diff
@@ -0,0 +1 @@
+from hs_atmosphere import * 
```

Comment:
In this file you should do something like this:

```python
import os

VALIIIC_data = os.path.join(__file__, 'VALIIIC.dat')
```

which will allow you to do:
```python
from pysac.mhs_atmosphere.hs_model import VALIIIC_data
```
to get the filename, same goes for the other data file.